src/Aspire.Cli/Commands/RunCommand.cs

```diff
@@ -40,8 +41,8 @@ public RunCommand(
         IConfiguration configuration,
         IDotNetSdkInstaller sdkInstaller,
         IFeatures features,
-        ICliUpdateNotifier updateNotifier
-        )
+        ICliUpdateNotifier updateNotifier,
+        IServiceProvider serviceProvider)
```

Comment:
Test IInteractionService for IExtensionInteractionService and then expose the backchannel as a property on IExtensionInteractionService.